src/SingleBookingPage/bookingTypes/Multicity.js

```diff
@@ -16,8 +17,18 @@ const Multicity = ({ booking }: Props) => (
     <AccordionTitle title="Trips" />
     {Array.isArray(booking.trips) &&
       booking.trips.map((trip, i) => (
-        // eslint-disable-next-line react/no-array-index-key
-        <Accordion key={i} trip={trip} />
+        <div
+          key={i} //eslint-disable-line react/no-array-index-key
+          onKeyUp={null}
+          role="button"
+          tabIndex="-1"
+          style={{ outline: 'none' }}
+          onClick={simpleTracker('smartFAQBookingOverview', {
+            action: 'openFlightCard',
+          })}
+        >
+          <Accordion trip={trip} />{' '}
```

Comment:
why `{' '}`?

Reply:
Absolute mystery, no idea. thx!

Reply:
This sends the event when clicking anywhere in the accordion, right? which is not 100 % same as clicking on accordion - you can click on the inner content as well and it's not "openFlightCard".